src/openrct2/cmdline_sprite.c

```diff
@@ -14,6 +14,9 @@
  *****************************************************************************/
 #pragma endregion
 
+#pragma warning(disable : 4706) // assignment within conditional expression
```

Comment:
I don't see where is this used, perhaps it's something that is only leftover from an earlier version? Can you please check if this is still required and where?